packages/openspp/src/Adaptor.js

```diff
@@ -660,6 +658,311 @@ export function unenroll(registrant_id, program_id) {
   };
 }
 
+/**
+ * create new individual for OpenSPP
+ * @public
+ * @example
+ * createIndividual({ name: "Individual 1" })
+ * @function
+ * @param {object} data - registrant create data
+ * @param {function} callback - An optional callback function
+ * @returns {Operation}
+ */
+export function createIndividual(data, callback=false) {
```

Comment:
If you default callback to a function which returns state:
```
export function createIndividual(data, callback=(s) => s) { /* ... */}
```

then later, instead of:

```
if (callback) {
    return callback(nextState);
}
return nextState;
```
You can simply do:
```
return callback(nextState)
```

Reply:
@josephjclark done fixing!